<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AstraViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_color">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>10</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 0);
border:0.5px solid black </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_depth">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 0);
border:0.5px solid black </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_ir">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>520</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 0);
border:0.5px solid black </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>520</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="camera">
    <property name="title">
     <string>    显示    </string>
    </property>
    <addaction name="action_show_depth"/>
    <addaction name="action_show_color"/>
    <addaction name="action_show_ir"/>
    <addaction name="action_show_points"/>
   </widget>
   <widget class="QMenu" name="save">
    <property name="title">
     <string>    保存    </string>
    </property>
    <addaction name="action_save_depth"/>
    <addaction name="action_save_color"/>
    <addaction name="action_save_ir"/>
    <addaction name="action_save_points"/>
   </widget>
   <addaction name="camera"/>
   <addaction name="save"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_show_color">
   <property name="text">
    <string>彩色图 </string>
   </property>
  </action>
  <action name="action_show_ir">
   <property name="text">
    <string>红外图</string>
   </property>
  </action>
  <action name="action_save_depth">
   <property name="text">
    <string>深度图 </string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="action_save_color">
   <property name="text">
    <string>彩色图</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="action_save_ir">
   <property name="text">
    <string>红外图</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="action_d2c">
   <property name="text">
    <string>深度图对齐</string>
   </property>
  </action>
  <action name="action_save_points">
   <property name="text">
    <string>点云</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="action_show_points">
   <property name="text">
    <string>点云</string>
   </property>
  </action>
  <action name="action_show_depth">
   <property name="text">
    <string>深度图</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
